Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1

```diff
@@ -8,10 +8,92 @@ Clear-Host
 [string] $server = 'solutionsnapshotapi.azurewebsites.net'
 
 [string] $CreateRelativityVersionAsyncUrl = "https://$($server)/api/external/CreateRelativityVersionAsync"
-[string] $ReadRelativityVersionAsyncUrl = "https://$($server)/api/external/ReadRelativityVersionAsync"
+[string] $updateApplicationVersionAsyncUrl = "https://$($server)/api/external/UpdateApplicationVersionAsync"
 
 $global:salesforceSessionObject = $null
 
+[string] $propogateCodingPostImportGuid = 'C0C7A2BE-12C9-49C0-BF3F-44EB85CFF3EA'
```

Comment:
I feel like these are two many variables. I think we can better organize them.

Can we create a PS class for storing the name, guid and version to update. Then create an array of this class and define the values there.

Later we could loop through the array to update each item. I am thinking similiar to what Scott did for NuGet packages. I'll send that link in slack.

PS Class:
- https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_classes?view=powershell-7&viewFallbackFrom=powershell-5.0
- https://stackoverflow.com/questions/17353797/how-to-initialize-an-array-of-custom-objects/17354800